PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Pull Request
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
While I'm not sure whether asking them to remove others is helpful or not (and therefor I can't say we should take that away), I do want to suggest we add an "agency" component to the instructions.

```suggestion
Please choose only the relevant options and remove the others.
```